docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# syntax=docker/dockerfile:1
+ARG NODE_VERSION=21.7.1
+
+FROM --platform=linux/amd64 node:${NODE_VERSION}-alpine
```

Comment:
The comment on `L1` causes BuildKit to pull the latest stable version of the Dockerfile syntax before the build. Just a precaution to use the same syntax on every build machine.

The `--platform=linux/amd64` restricts the build environment. I think we would only see the build fail if we were doing some CPU architecture specific operations, and in that case it would make a difference if we build on `linux/amd64` or `linux/arm64`. 

We can leave it, delete it, in the current context it makes no difference.
I am going to remove it for simplicity and move on.